libs/server/Resp/Objects/SortedSetCommands.cs

```diff
@@ -879,5 +879,315 @@ private unsafe bool SortedSetDifferenceStore<TGarnetApi>(ref TGarnetApi storageA
 
             return true;
         }
+
+        /// <summary>
+        /// Computes an intersection operation between multiple sorted sets
+        /// and returns the result to the client.
+        /// </summary>
+        /// <param name="storageApi"></param>
+        /// <returns></returns>
+        private unsafe bool SortedSetIntersect<TGarnetApi>(ref TGarnetApi storageApi)
+            where TGarnetApi : IGarnetApi
+        {
+            if (parseState.Count < 1)
+            {
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.ZINTER));
+            }
+
+            // Number of keys
+            if (!parseState.TryGetInt(0, out var nKeys))
+            {
+                while (!RespWriteUtils.WriteError(CmdStrings.RESP_ERR_GENERIC_VALUE_IS_NOT_INTEGER, ref dcurr, dend))
+                    SendAndReset();
+                return true;
+            }
+
+            if (parseState.Count < nKeys + 1)
```

Comment:
Another check necessary - if (nKeys < 1), you need to return `ERR at least 1 input key is needed for 'zinter' command`